src/analyzer/move2_mt.R

```diff
@@ -0,0 +1,122 @@
+library(logger)
+library(move)
+library(move2)
```

Comment:
add new library to `renv` (TODO for Clemens)

Reply:
@andreakoelzsch 

what do I have to do to install `move2`?

```
> install.packages('move2')
Error: package 'move2' is not available
```

Reply:
install_gitlab("bartk/move2")